### PR TITLE
Good snippet

### PR DESCRIPTION
**Pull request by Tomas2h**
Changed some snippet classes and now fixed the checkstyle error and added some java doc

### CODE REVIEW

File: test/TimedSnippetTest.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  }
+
+  /**
+   * Does some clean up for the test.
+   * 
+   * @throws Exception
+   *           If the clean up fails.
+   */
+  @After
+  public void tearDown() throws Exception {
```
Comment (daanschipper) [inline test/TimedSnippetTest.java]: Empty method

**Comment by daanschipper** [comment]
If you do a quick refactoring of the code i'll accept your pull request.

**Comment by Tomas2h** [comment]
Fixed the last errors

**Comment by daanschipper** [comment]
Nice implementation of the comments as object, viable piece of the snippet selection.
Please use better commits messages the next time.

**Comment by StefanBoodt** [comment]
The update of the javadoc seems nice and some of the CheckStyle errors have been removed. However you did walk away from the java formatting convention of using a tab character to indent and instead went with 2 spaces, as is default for the C programming language.

I know this is the fault of CheckStyle but that kind of changes is not necessary and it's not the way to do this, because CheckStyle does not follow the convention. I sincerely think this pull request should be reviewed by someone else as well as I am a little biased on the topic of some of these files.

**Comment by StefanBoodt** [comment]
If you want to know how to fix the situation you seem to have created at your own local computer by doing CheckStyle -> Create formatter profile you should go to (Window -> ) Preferences -> Java -> Code style -> Formatter  and then do restore defaults.

**Comment by daanschipper** [comment]
This branch is not yet finished, it needs more work done to it to comply to our definition of done and has some issues. As you can also see the build fails

**Pull request closed without merging**